tests/unit/evidence_lm_test.py

```diff
@@ -930,12 +936,11 @@ def test_moving_off_object(self):
         self.assertEqual(
             num_matching_steps,
             sum(
-                exp.model.learning_modules[0].buffer.features["patch"][
-                    "on_object"
-                ][:num_matching_steps]
+                exp.model.learning_modules[0].buffer.features["patch"]["on_object"][
+                    :num_matching_steps
```

Comment:
When I first read this, my brain said it was a symbol like in Ruby or Elixir or a keyword in Lisp. That wouldn't happen if the line weren't being wrapped like it is.

I generally don't like how Ruff wraps lines like this, but this one is especially bothersome. We might want to consider making a variable for `exp.model.learning_modules[0]` above the asserts before this one and shortening all the places it's used.

Reply:
I removed extraneous style changes from the PR. It's back to where it was.